built_value_generator/lib/src/serializer_source_class.dart

```diff
@@ -330,7 +330,7 @@ class $serializerImplName implements StructuredSerializer<$genericName> {
   @override
   $genericName deserialize(Serializers serializers, Iterable<Object$orNull> serialized,
       {FullType specifiedType = FullType.unspecified}) {
-    ${_generateGenericsSerializerPreamble()}
+    ${_generateGenericsSerializerPreamble()} ${_generateHaveSetValueDeclarations()}
```

Comment:
I have put some of these new snippet generators on the same lines to reduce whitespace diffs. I welcome any suggestions for ways to clean this up.

Reply:
I'm not too worried about either the layout here or the generated output; the generated output does usually change a lot with generator changes. Thanks.